File: AngularBoot/src/app/pages/_models/shop/daoShopLocation.ts
export class DaoShopLocation {

    private type: string;
    private coordinates: any[];

    public getType(): string {
        return this.type;
    }

    public setType(type: string) {
        this.type = type;
    }

    public getCoordinates(): any[] {
        return this.coordinates;
    }

    public setCoordinates(coordinates: any[]) {
        this.coordinates = coordinates;
    }
}

File: SpringBoot/shop-founders-modules/shop-founders-core/src/main/java/hf/shopfounders/model/DaoShopLocation.java
package hf.shopfounders.model;

public class DaoShopLocation {

    private String type;
    private double[] coordinates;

    public DaoShopLocation() {
    }

    public DaoShopLocation(String type, double[] coordinates) {
        this.type = type;
        this.coordinates = coordinates;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public double[] getCoordinates() {
        return coordinates;
    }

    public void setCoordinates(double[] coordinates) {
        this.coordinates = coordinates;
    }

    @Override
    public String toString() {
        return String.format(
                "Location[coordinates='%s', coordinates='%s']",
                coordinates[0], coordinates[1]);
    }
}
